vignettes/RPyGeo.Rmd

```diff
@@ -93,17 +93,18 @@ To change the default scratch workspace location, use parameter `scratch_workspa
 
 ### Find out about ArcPy functionality
 
-If you do not know the name of the ArcPy function or class, the `rpygeo_search` function can be used. 
+<!-- Marc, can we reference the arcpy function list/help page here. What happens if the search_term = "" -> does this return all available functions? -->
```

Comment:
Can `rpygeo_search()` also return all available functions?

Reply:
`rpygeo_search("", arcpy)` returns all available functions and classes of the main module. Unfortunately, not the functions of the other modules and extensions (e.g. `arcpy$sa` or `arcpy$mapping`) This would be possible if we map the module names and their module abbreviations by hand (e.g. "Spatial" = "sa").

Reply:
Ok, I suggest to replace the default value (NULL) of `search_term` by "" (have already done so in a commit, if you don't like it you can still revert).